datamodel/changelogs/0001/05_add_vl_pipe_section_pipe_dn.sql

```diff
@@ -0,0 +1,148 @@
+-- Change type of column -- see https://stackoverflow.com/questions/7162903/how-to-alter-a-columns-data-type-in-a-postgresql-table (version that allows to have already data in db)
+
+--ALTER TABLE tdh_od.pipe_section ALTER COLUMN nominal_diameter TYPE integer USING (NULLIF(nominal_diameter, '')::integer);
+
+ALTER TABLE tdh_od.pipe_section
+DROP COLUMN IF EXISTS nominal_diameter;
+
+ALTER TABLE tdh_od.pipe_section ADD COLUMN nominal_diameter integer;
+ /*---------------------------------------------------------------------------------------*/
+/* AJOUT DE LA TABLE DES DIAMETRES NOMINAUX DE CONDUITES () */
+/*---------------------------------------------------------------------------------------*/
+
+CREATE TABLE tdh_vl.pipe_section_pipe_dn () INHERITS (tdh_vl.value_list_base);
```

Comment:
```suggestion
CREATE TABLE tdh_vl.pipe material () INHERITS (tdh_vl.value_list_base);
```

Reply:
Should be tdh_vl.classname_attributname - so tdh_vl.pipe_section_mg_material